Servers/UI/OJS.Servers.Ui/ClientApp/src/components/diff/Diff.module.scss

```diff
@@ -1,4 +1,4 @@
-@use 'src/styles/variables';
+@use '../../styles/variables';
```

Comment:
Why make this change? It is not done in Servers/UI/OJS.Servers.Ui/ClientApp/src/components/dropdown/Dropdown.module.scss for example

Reply:
this was unintenional, reverting it

Reply:
done